template_config.yaml

```diff
@@ -22,6 +22,18 @@ tag_operation_files: # DICOM tag anonymisation operations
         - "base-tag-operations.yaml" # Base schema
     manufacturer_overrides: none # Manufactuer-dependendent overrides
 
+allowed_manufacturers:
+  - regex: ".*"  # allow all manufacturers
+    exclude_series_numbers: []
```

Comment:
I think our template shouldn't allow all through, because that opens us up to exporting series which are burned in. But adding in canon's 8000 series to exclude is a good idea